### PR TITLE
GPG toolstack upgrade to latest available versions (Fixes Gawk issue)

### LINKED ISSUE #668: libgpg-error build fails with newer gawk (Upstream Issue)
Compilation fails upon libgpg-error when using gawk 5.0+ OpenWRT fixed this using commit https://github.com/openwrt/packages/pull/9371/commits/9afde0d39ae364c327063d36b188465058056a86 and this should give insight into the fixes needed until upstream resolves this.

Log: https://pastebin.com/raw/F8987WAg

Tested on Fedora 31 fully updated 30/01/2020
Gawk version which failed: 5.0.1 (gawk-5.0.1-5.fc31.x86_64.rpm)
Solution was to forcibly install Gawk from Fedora 30 - Gawk 4.2.1 (gawk-4.2.1-6.fc30.x86_64.rpm)
`rpm -Uvh https://download-ib01.fedoraproject.org/pub/fedora/linux/releases/30/Everything/x86_64/os/Packages/g/gawk-4.2.1-6.fc30.x86_64.rpm --force`

### PR DESCRIPTION
**Pull request by tlaurion**
This includes to be reverted CircleCi config to [test current build over debian bullseye which was successful](https://app.circleci.com/pipelines/github/tlaurion/heads/144/workflows/4542964a-8a00-4951-858b-093f4de9ff39/jobs/154), while resulting build is currently untested.

Fixes #668 and problem in #710 PR to include Heads into Guix facing gawk issues

Todo: push additional commit to reverse that changeset once reviewed.

Reviewers: 
I repeat here message in [commit](https://github.com/tlaurion/heads/commit/b52ed555220aabd2f9328db1a8ac913d488b3968)
```
user@x230-master:~/heads$ grep -Rn unrecognized build/log/
build/log/libusb.configure.log:2:configure: WARNING: unrecognized options: --disable-tests
build/log/libusb.configure.log:157:configure: WARNING: unrecognized options: --disable-tests
build/log/libgpg-error.configure.log:2:configure: WARNING: unrecognized options: --disable-asm
build/log/libgpg-error.configure.log:223:configure: WARNING: unrecognized options: --disable-asm
build/log/pinentry.configure.log:2:configure: WARNING: unrecognized options: --disable-modular-tests, --disable-qt, --disable-ft, --disable-fc, --disable-qt, --disable-gtk-doc, --disable-cups
build/log/pinentry.configure.log:134:configure: WARNING: unrecognized options: --disable-modular-tests, --disable-qt, --disable-ft, --disable-fc, --disable-qt, --disable-gtk-doc, --disable-cups
build/log/gpg2.configure.log:2:configure: WARNING: unrecognized options: --with-libusb, --disable-asm, --disable-keyserver-helpers, --disable-hkp, --disable-finger, --disable-dns-srv, --disable-dns-cert, --disable-wks-server, --disable-static
build/log/gpg2.configure.log:376:configure: WARNING: unrecognized options: --with-libusb, --disable-asm, --disable-keyserver-helpers, --disable-hkp, --disable-finger, --disable-dns-srv, --disable-dns-cert, --disable-wks-server, --disable-static
build/log/libksba.configure.log:2:configure: WARNING: unrecognized options: --disable-nls, --disable-asm
build/log/libksba.configure.log:147:configure: WARNING: unrecognized options: --disable-nls, --disable-asm
build/log/libassuan.configure.log:2:configure: WARNING: unrecognized options: --disable-nls, --disable-asm
build/log/libassuan.configure.log:181:configure: WARNING: unrecognized options: --disable-nls, --disable-asm
build/log/npth.configure.log:2:configure: WARNING: unrecognized options: --disable-nls, --with-libgpg-error-prefix, --disable-asm
build/log/npth.configure.log:146:configure: WARNING: unrecognized options: --disable-nls, --with-libgpg-error-prefix, --disable-asm

left alone, while not doing anything as of now:
--with-libgpg-error-prefix
--with-libusb="$(INSTALL)"

TODO: Revisit gpg2 toolstack and remove unneeded options and revise the two configure statements above.
```

### CODE REVIEW

**Comment by tlaurion** [comment]
@MrChromebox Any opposition?

File: config/coreboot-x230.config [hunk 1]
```diff
@@ -1,8 +1,6 @@
 CONFIG_LOCALVERSION="heads"
 CONFIG_ANY_TOOLCHAIN=y
-# CONFIG_INCLUDE_CONFIG_FILE is not set
-# CONFIG_COLLECT_TIMESTAMPS is not set
-CONFIG_USE_BLOBS=y
+CONFIG_USE_BLOBS=n
```
Comment (MrChromebox) [inline config/coreboot-x230.config:3]: these changes seem unrelated

**Comment by MrChromebox** [comment]
@tlaurion let me give this a test tomorrow, looks good at first glance outside of the unrelated x230 config change

File: modules/pinentry [hunk 2]
```diff
@@ -21,12 +21,13 @@ pinentry_configure := ./configure \
...
 	--disable-fallback-curses \
 	--disable-pinentry-curses \
-	--disable-pinentry-qt \
 	--disable-pinentry-gtk2 \
 	--disable-pinentry-gnome3 \
 	--disable-pinentry-fltk \
 	--disable-pinentry-emacs \
 	--disable-fallback-curses \
+	--disable-pinentry-qt5   \
+	--disable-pinentry-fltk  \
```
Comment (paulmenzel) [inline modules/pinentry:30]: This appears twice now.

File: modules/pinentry [hunk 3]
```diff
@@ -21,12 +21,12 @@ pinentry_configure := ./configure \
 	--disable-libsecret \
 	--disable-fallback-curses \
 	--disable-pinentry-curses \
-	--disable-pinentry-qt \
```
Comment (MrChromebox) [inline modules/pinentry:24]: why is this being removed? Doing so seems to introduce a dependency on Qdialog which we're not providing

**Comment by tlaurion** [comment]
only configure warning still being present is `--with-libusb` in gpg2 module.

**Comment by MrChromebox** [comment]
built for librem13v4, flashed, booted, performed a factory reset and everything went smoothly. LGTM now
